src/main/groovy/org/boozallen/plugins/jte/init/PipelineDecorator.groovy

```diff
@@ -144,7 +146,7 @@ class PipelineDecorator extends InvisibleAction {
             FileSystemWrapper fs = FileSystemWrapper.createFromJob(flowOwner)
             String repoJenkinsfile = fs.getFileContents("Jenkinsfile", "Repository Jenkinsfile", false)
             if (repoJenkinsfile){
-                Boolean allowScmJenkinsfile = pipelineConfig.containsKey("allow_scm_jenkinsfile") ? pipelineConfig.allow_scm_jenkinsfile : true
+                Boolean allowScmJenkinsfile = jteBlockWrapper.allow_scm_jenkinsfile
```

Comment:
i believe i only set this variable for readability as to not do this logic within the if statement condition. 

now that it's been abstracted into the jteBlockWrapper, we can probably do:

```groovy
if(jteBlockWrapper.allow_scm_jenkinsfile){
  return repoJenkinsfile
}
```

i think?

Reply:
yeah, let me do that